rpcserver.go

```diff
@@ -2172,28 +2172,24 @@ func (r *rpcServer) parseOpenChannelReq(in *lnrpc.OpenChannelRequest,
 
 	var feeRate chainfee.SatPerKWeight
 
-	// Skip estimating fee rate for PSBT funding.
-	if in.FundingShim == nil || in.FundingShim.GetPsbtShim() == nil {
```

Comment:
not really sure why this was added, pre this PR we would only determine fee rates for `OpenChannelSync`/`OpenChannel` requests, right? noticing that there isn't itest coverage of `OpenChannelSync`

Reply:
It's introduced in this [commit](https://github.com/lightningnetwork/lnd/pull/8667/commits/61e9c11881cfc1464233a866fbce516d94e35beb) but I forgot about the context🤦🏻 Think it's due to PSBT having its own fee estimation logic but couldn't be sure.

Reply:
Added an itest for the OpenChannelSync flow.